src/cache/item.rs

```diff
@@ -17,6 +17,13 @@ impl<T> Item<T> {
             .map(|expiry| expiry < Instant::now())
             .unwrap_or(false)
     }
+
+    pub fn elapsed(&self) -> u128 {
+        self.expiry
+            .map(|expiry| Instant::now() - expiry)
```

Comment:
`expiry` is `now + duration` , will it cause a `overflow` here?
say, at 13:00:00 set a expiry after 10 mins, i.e. expiry is at 13:10:00
and check at 13:05:00, then that will be `13:05:00 - 13:10:00` ?

Reply:
Yeah, that could overflow. WIll fix.